specification/communication/data-plane/SipRouting/preview/2024-11-15-preview/communicationservicessiprouting.json

```diff
@@ -166,8 +166,7 @@
     "Trunk": {
       "description": "Represents a SIP trunk for routing calls. See RFC 4904. Can be expanded with additional data.",
       "required": [
-        "sipSignalingPort",
-        "enabled"
```

Comment:
This was incorrectly marked as required in the previous preview version. It cannot be required as it would introduce breaking change against the latest stable build.